dipy/workflows/tracking.py

```diff
@@ -6,18 +6,60 @@
 
 from nibabel.streamlines import save, Tractogram
 
-from dipy.direction import DeterministicMaximumDirectionGetter
+from dipy.direction import (DeterministicMaximumDirectionGetter,
+                            ProbabilisticDirectionGetter,
+                            ClosestPeakDirectionGetter)
 from dipy.io.image import load_nifti
 from dipy.io.peaks import load_peaks
 from dipy.tracking import utils
-from dipy.tracking.local import (ThresholdTissueClassifier,
-                                 LocalTracking)
+from dipy.tracking.local import (ThresholdTissueClassifier, LocalTracking,
+                                 CmcTissueClassifier,
+                                 ParticleFilteringTracking)
 from dipy.workflows.workflow import Workflow
 
 
-class GenericTrackFlow(Workflow):
+class LocalFiberTrackingPAMFlow(Workflow):
+    @classmethod
+    def get_short_name(cls):
+        return 'lf_track'
```

Comment:
to be consistent with the workflow name change, I guess this should be 'local_track'?